src/pages/UpdateProduct.js

```diff
@@ -2,72 +2,23 @@ import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 import './styles.css';
 
-const UpdateProduct = ({ productId }) => {
-  const [name, setName] = useState('');
-  const [price, setPrice] = useState('');
-  const [description, setDescription] = useState('');
-  const [category, setCategory] = useState('');
-  const [company, setCompany] = useState('');
-  const [featured, setFeatured] = useState(false);
-  const [freeShipping, setFreeShipping] = useState(false);
-  const [inventory, setInventory] = useState(15);
+const UpdateProduct = ({ product, onUpdate }) => {
```

Comment:
This component is loaded from the `<SingleProductPage>` component. That component has already fetched the product info that we need here. Instead of making this component _also_ make a call to the backend, it would be better to just pass the product down to this component via the props.